tests/jwk_test.py

```diff
@@ -139,101 +151,114 @@ def test_to_partial_json(self):
 
     def test_from_json(self):
         from josepy.jwk import JWK
-        self.assertEqual(
-            self.jwk256, JWK.from_json(self.jwk256json))
-        self.assertEqual(
-            self.jwk512, JWK.from_json(self.jwk512json))
+
+        self.assertEqual(self.jwk256, JWK.from_json(self.jwk256json))
+        self.assertEqual(self.jwk512, JWK.from_json(self.jwk512json))
         self.assertEqual(self.private, JWK.from_json(self.private_json))
 
     def test_from_json_private_small(self):
         from josepy.jwk import JWK
+
         self.assertEqual(self.private, JWK.from_json(self.private_json_small))
 
     def test_from_json_missing_one_additional(self):
         from josepy.jwk import JWK
-        del self.private_json['q']
+
+        del self.private_json["q"]
         self.assertRaises(errors.Error, JWK.from_json, self.private_json)
 
     def test_from_json_hashable(self):
         from josepy.jwk import JWK
+
         hash(JWK.from_json(self.jwk256json))
 
     def test_from_json_non_schema_errors(self):
         # valid against schema, but still failing
         from josepy.jwk import JWK
-        self.assertRaises(errors.DeserializationError, JWK.from_json,
-                          {'kty': 'RSA', 'e': 'AQAB', 'n': ''})
-        self.assertRaises(errors.DeserializationError, JWK.from_json,
-                          {'kty': 'RSA', 'e': 'AQAB', 'n': '1'})
+
+        self.assertRaises(
+            errors.DeserializationError, JWK.from_json, {"kty": "RSA", "e": "AQAB", "n": ""}
+        )
+        self.assertRaises(
+            errors.DeserializationError, JWK.from_json, {"kty": "RSA", "e": "AQAB", "n": "1"}
+        )
 
     def test_thumbprint_go_jose(self):
         # https://github.com/square/go-jose/blob/4ddd71883fa547d37fbf598071f04512d8bafee3/jwk.go#L155
         # https://github.com/square/go-jose/blob/4ddd71883fa547d37fbf598071f04512d8bafee3/jwk_test.go#L331-L344
         # https://github.com/square/go-jose/blob/4ddd71883fa547d37fbf598071f04512d8bafee3/jwk_test.go#L384
         from josepy.jwk import JWKRSA
-        key = JWKRSA.json_loads("""{
+
+        key = JWKRSA.json_loads(
+            """{
     "kty": "RSA",
     "kid": "bilbo.baggins@hobbiton.example",
     "use": "sig",
-    "n": "n4EPtAOCc9AlkeQHPzHStgAbgs7bTZLwUBZdR8_KuKPEHLd4rHVTeT-O-XV2jRojdNhxJWTDvNd7nqQ0VEiZQHz_AJmSCpMaJMRBSFKrKb2wqVwGU_NsYOYL-QtiWN2lbzcEe6XC0dApr5ydQLrHqkHHig3RBordaZ6Aj-oBHqFEHYpPe7Tpe-OfVfHd1E6cS6M1FZcD1NNLYD5lFHpPI9bTwJlsde3uhGqC0ZCuEHg8lhzwOHrtIQbS0FVbb9k3-tVTU4fg_3L_vniUFAKwuCLqKnS2BYwdq_mzSnbLY7h_qixoR7jig3__kRhuaxwUkRz5iaiQkqgc5gHdrNP5zw",
+    "n": n4EPtAOCc9AlkeQHPzHStgAbgs7bTZLwUBZdR8_KuKPEHLd4rHVTeT-O-XV2jRojdNhxJWTDvNd7nqQ0VEiZQHz_AJmSCpMaJMRBSFKrKb2wqVwGU_NsYOYL-QtiWN2lbzcEe6XC0dApr5ydQLrHqkHHig3RBordaZ6Aj-oBHqFEHYpPe7Tpe-OfVfHd1E6cS6M1FZcD1NNLYD5lFHpPI9bTwJlsde3uhGqC0ZCuEHg8lhzwOHrtIQbS0FVbb9k3-tVTU4fg_3L_vniUFAKwuCLqKnS2BYwdq_mzSnbLY7h_qixoR7jig3__kRhuaxwUkRz5iaiQkqgc5gHdrNP5zw",  # noqa
```

Comment:
🤨 I assume (and hope) `black` was not responsible for this quote disappearing, haha.

Reply:
Not really sure what happened. I had some issues with flake8, and tried moving the `# noqa` marker. Anyway, it's fixed/reverted now :)